core/test/cosmos/transactions_test.cpp

```diff
@@ -77,8 +77,8 @@ TEST(CosmosTransactionTest, BuildSignedSendTxForBroadcast) {
     EXPECT_EQ(sendMessage.amount.front().amount, "1000000");
     EXPECT_EQ(sendMessage.amount.front().denom, "uatom");
 
-    const std::string expected = "{\"mode\":\"block\",\"tx\":" + strTx + "}";
-    EXPECT_EQ(tx->serializeForBroadcast(), expected);
+    const std::string expected = "{\"mode\":\"async\",\"tx\":" + strTx + "}";
```

Comment:
How do you chose block | sync | async here for each of the message types? (Just for my curiosity)

Reply:
just randomly to have all the types we currently test :D